test/iron-input.html

```diff
@@ -107,162 +123,234 @@
   <script>
 
     suite('basic', function() {
+      test('can initialize programmatically', function(done) {
+        var ironInput = fixture('programmatic');
+        assert.isNull(ironInput.querySelector('input'));
 
-      test('setting bindValue sets value', function() {
-        var input = fixture('basic');
-        input.bindValue = 'foobar';
-        assert.equal(input.value, input.bindValue, 'value equals to bindValue');
+        ironInput.appendChild(document.createElement('input'));
+        var input = ironInput.querySelector('input');
+        assert.ok(input);
+
+        // Mutation observer is async, so wait one tick.
+        Polymer.Base.async(function() {
+          assert.ok(ironInput._inputElement);
```

Comment:
If we are going to reach for the private / protected property in a test, maybe we should have a public accessor for this thing?